Services/DataBaseService/Contollers/DataBaseController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using DTO;
+using DataBaseService.Interfaces;
+
+namespace DataBaseService.Contollers
+{
+    [ApiController]
+    [Route("[controller]")]    
+    public class DataBaseController : ControllerBase
+    {
+        [Route("CreateUser")]
```

Comment:
How about just create? I think we should have common style?

Reply:
> How about just create? I think we should have common style?

Idk, url like .../database/create might not be really obvious, we can create many different things in database